hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieHiveCatalog.java

```diff
@@ -477,6 +478,8 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
       if (!ignoreIfExists) {
         throw new TableAlreadyExistException(getName(), tablePath, e);
       }
+    } catch (HoodieValidationException e) {
+      throw e;
     } catch (Exception e) {
       throw new HoodieCatalogException(
```

Comment:
Shouldn't any exception been caught by the `Exception` ?

Reply:
@danny0405  yes, but I throws a HoodieValidationException, maybe i need to fix ut to assert throw a HoodieCatalogException

Reply:
> maybe i need to fix ut to assert throw a HoodieCatalogException

You should.